### ISSUE TITLE
"Check for updates" doesn't find the 1.5.x updates

### ISSUE THREAD

**Issue by abh** [coreos/tectonic-forum#76]
### Tectonic Version 

1.4.7-tectonic.1

### Environment

Bare metal on KVM

### Expected Behavior

Offer to upgrade to 1.5.x.

### Actual Behavior

The UI says it's "Up to date".

I've been logging in logs with journalctl on all the nodes and a bunch of pods without finding anything relevant. I also used tcpdump on the controller and a couple of the masters while clicking the "check for update" button to see if any of the hosts tried something that didn't work (I didn't find anything interesting).

**Comment by mfburnett** [comment]
Hi @abh, thanks for filing this issue. As Operators are still experimental, we're not planning on publishing a supported upgrade path for 1.4.x to 1.5.x or 1.5.x to 1.6.x. We do hope to be able to publish an upgrade from 1.6.x to 1.7.x, however, as Operators move to GA!

**Comment by abh** [comment]
Ok, in that case it'd be nice to add to the documentation that upgrading between 1.4->1.5->1.6 requires a re-install (or whatever the workaround is). The UI could possibly also make that more clear rather than just saying "all is up to date". :-)

**Comment by praseodym** [comment]
I also did not see an update from 1.5.2-tectonic.2 to 1.5.3-tectonic.1.

**Comment by mfburnett** [comment]
@abh definitely: will make sure we add this to docs. @praseodym we changed the channel names for 1.5.3-tectonic.1, so updating requires some extra steps - going to add in our docs in a few days but tomorrow I will make sure to post instructions here!

**Comment by mfburnett** [comment]
@praseodym for 1.5.2-tectonic.2, we just pushed an update payload for 1.5.4-tectonic.1 (released today). Instructions are as follows:

Tectonic recently modified the naming scheme of the automatic update channels. The existing channel (“alpha”) has been deprecated, and we are now in the process of rolling out new update channels based on the Tectonic/Kubernetes minor release versions.

When you launched your cluster, the update channel was still set to “alpha”, however to receive new update payloads you will need to change this to the latest Tectonic update channel (“tectonic-1.5”).

To do this you will need to modify a ThirdPartyResource object to reference the new channel. Using `kubectl` this can done by running:

```sh
$ kubectl replace -f -<<EOF
apiVersion: coreos.com/v1
appID: 6bc7b986-4654-4a0f-94b3-84ce6feb1db4
automaticUpdate: false
channel: tectonic-1.5
kind: ChannelOperatorConfig
metadata:
  name: default
  namespace: tectonic-system
server: https://tectonic.update.core-os.net
triggerUpdate: false
triggerUpdateCheck: false
updateCheckInterval: 2700
EOF
```

Your cluster should now be subscribed to the “tectonic-1.5” channel, this can be confirmed by going to the Cluster Settings screen and checking the Channel section.

To check for an update manually, click the “Check for Updates” button in the console UI under Cluster Settings.

When an update is available, the button will turn green and read “Start Update”. Clicking the button will trigger the upgrade.

You should see this after successfully updating your cluster:
<img width="612" alt="screen shot 2017-03-17 at 3 37 31 pm" src="https://cloud.githubusercontent.com/assets/6687345/24065378/ab9b138a-0b27-11e7-9d39-cdae7cd16906.png">

**Comment by praseodym** [comment]
This worked, thanks for your help!

**Comment by InAnimaTe** [comment]
This is great news. @mfburnett any chance this method would also work for `1.5.2-tectonic.1`?

**Comment by joshix** [comment]
https://coreos.com/tectonic/docs/1.5.2-tectonic.2/admin/upgrade.html#upgrading-tectonic--kubernetes

**Comment by mfburnett** [comment]
@InAnimaTe unfortunately no, this only works for 1.5.2-tectonic.1. Good news is that we've been able to successfully update 1.5.3-tectonic.1 and 1.5.4-tectonic.1 via the UI to 1.5.5-tectonic.2!
